tests/test_scripts.py

```diff
@@ -309,47 +324,39 @@ def _check_train_ex_result(result: dict):
         ["train.normalize_disable", "reward.normalize_input_disable"],
     ),
 )
-def test_train_adversarial(tmpdir, named_configs):
+@pytest.mark.parametrize("command", ("airl", "gail"))
```

Comment:
:) yes we should test both

Reply:
I really didn't expect the problem from airl lol...